Modules/Module_Processing/Module_Population/xASL_im_CreateAnalysisMask.m

```diff
@@ -49,10 +49,11 @@
 bSkipStandard = 0;
 
 if x.dataset.nSubjectsSessions<16
-    fprintf('%s\n', ['Too few subjects (' num2str(x.dataset.nSubjectsSessions) ') to create population-based analysis mask']);
-    x.S.MaskSusceptibility = xASL_im_IM2Column(ones(121,145,121),x.S.masks.WBmask);
-    x.S.VBAmask = x.S.MaskSusceptibility;
-    bSkipStandard = 1;
+    % With too small datasets, created templated are not reliable
+    fprintf('\n\n%s\n\n', ['Only n=' num2str(x.dataset.nSubjectsSessions) ' subject*runs, population-based analysis mask may not be useful']);

```

Comment:
It might be easier to create the susceptibility mask so that we have it in all cases. No?

Reply:
For clarification:

1) `xASL_module_ASL` -> `xASL_wrp_CreateAnalysisMask` now always creates all masks for all ASL sessions, for 3D GRASE this will be a dummy brain mask
2) `xASL_module_Population` -> `xASL_wrp_CreatePopulationTemplates` creates average template images of all NIfTIs in standard space, including all masks
2) `xASL_module_Population` -> `xASL_im_CreateAnalysisMask` takes the mask templates (of FoV, MaskSusceptibility, MaskVascular, etc) and generates a group analysis mask from them. Checking that at least 95% of the population have a certain voxel to include it, etc.

Does it help if I rename:

`xASL_im_CreateAnalysisMask` -> `xASL_im_CreateGroupAnalysisMask`
?

So instead of creating an empty group susceptibility mask that we have it in all cases, with my new changes, we will here now skip this, because we will always use the individual susceptibility masks.

Reply:
I did the rename as suggested. And agree with the rest.